common/src/main/java/io/pravega/common/Exceptions.java

```diff
@@ -212,6 +212,18 @@ public static void checkNotClosed(boolean closed, Object targetObject) throws Ob
         }
     }
 
+    /**
+     * Logs the exception and its cause to error log.
+     *
+     * @param t   The exception to be logged.
+     */
+    public static void unwrapAndLogException(Throwable t) {
+       log.error("Exception :", t);
```

Comment:
This is a nice utility, however we lose some valuable information by doing so. When you invoke `log.error` (or any other method), this will be recorded as coming from `Exceptions` and not from whatever class is invoking this (we also lose the line number as well).

Reply:
Dropping this method as `log.error` or `warn` itself goes through the chains.